src/main/java/xyz/nucleoid/plasmid/mixin/game/event/ExplosionMixin.java

```diff
@@ -30,7 +30,7 @@ private void affectWorld(boolean blocks, CallbackInfo ci) {
         ManagedGameSpace gameSpace = ManagedGameSpace.forWorld(this.world);
         if (gameSpace != null) {
             try {
-                gameSpace.invoker(ExplosionListener.EVENT).onExplosion(this.affectedBlocks);
+                gameSpace.invoker(ExplosionListener.EVENT).onExplosion(this);
```

Comment:
On second thought, is `affectedBlocks` exposed and accessible to users?

Reply:
Yes, there is a public getter method available. I realized, though, that the `blocks` (actually `particles`) parameter is not passed and the type is incorrect (`ExplosionMixin` rather than `Explosion`).